src/Types.re

```diff
@@ -0,0 +1,23 @@
+// all operations found in `/amazon-cognito-identity-js/src/CognitoUser.js` except `SignUp` which is in `CognitoUserPool.js`;
+[@bs.deriving jsConverter]
```

Comment:
:+1:

Reply:
That comment is really helpful can you add in an actual link to github to make it easier to cross reference?

Reply:
As in, fully qualified, `https://` ...